### ISSUE TITLE
[Enhancement] Texture Filtering

### ISSUE THREAD

**Issue by shinra358** [PeterTh/gedosato#201]
Since AA only deals with edges, I would love an option for games that have not so pretty textures to be enhanced by texture filtering similar to what emulators can do. Old games and some new games have this problem because these games lack graphical options. For example, Final Fantasy III's horrific textures or any Square game that's coming str8 from Android, older 2D games, games with good 3D but low rezzed pixelated text, etc etc. FFXIII could even use a little with it's text.

**Comment by DangusTaargus** [comment]
I came in here specifically to ask this question in regards to the Steam versions of Final Fantasy III, IV and IV The After Years. Games which apply no texture filtering to their textures, but which have had this feature working via PPSSPP. Is there any way to add this feature to allow users to choose from a handful of texture filters, or even just Bilinear for now, to clean the game up a bit?

**Comment by PeterTh** [comment]
Well, since this request was opened a long time ago I actually implemented some texture filtering for DX11. I'd need to backport that to DX9 and make it more general which is not trivial.

**Comment by shinra358** [comment]
So far, opengl games seem to need it more. Haven't seen alot of ashyness in dx11 games that I know of.

**Comment by DangusTaargus** [comment]
Yeah I think all of the games I was referring to use openGL. But either way, I'm sure it's something people would appreciate in regards to the final fantasy titles. If you can make it work that'd be amazing, but if not, I understand.
